### ISSUE TITLE
I have a tab can't close

### ISSUE THREAD

**Issue by coder-free** [a85/POSTMan-Chrome-Extension#10634]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

I have a empty tab can't close.

### Steps To Reproduce

I don't know. I have many tab not close. Today I want close all tabs. I found there has a tab is empty and can't close.

### Screenshots or Videos

https://user-images.githubusercontent.com/2630290/152564498-9349b00d-5f05-4c1c-830f-913302463a02.mp4

### Environment Information

```markdown
- Operating System: macOS 12.1
- Platform Type: Native App
- Postman Version: 9.11.0
```

**Comment by coder-free** [comment]
This issues is duplicate with #10622.

**Comment by postman-staff** [comment]
Hi @coder-free

Sorry to see you're encountering the issue.

We are aware of this case and the team is working on resolving the same. To let you know, the issue branched from the previous case where there were empty ids that can be cleared to resolve the same from your end. The resolution for the action would be:


- Signing out of your account
- Clearing cache as [per the article here](https://support.postman.com/hc/en-us/articles/4410785125271)
- Re-logging back to your account

These steps would completely clear out these cases here.
Let us know how it goes and we will take it from there.

via @Raj-postman | Postman Support

**Comment by postman-staff** [comment]
@coder-free I just wanted to follow up here and see if the provided resources helped or if you still needed assistance. Thank you!

via @Raj-postman | Postman Support

**Comment by coder-free** [comment]
@postman-support Yes, I still need assistance. My empty tab still can't be close.

**Comment by postman-staff** [comment]
@coder-free Could you check if you are able to see the same issue in this workspace even on Postman Web: https://go.postman.co/home?

If yes, please submit a support request from this link here: https://www.postman.com/support to help us check this further in terms of your account. Sharing this GitHub link in the ticket will help us proceed quickly from there.

via @Raj-postman | Postman Support

**Comment by coder-free** [comment]
@postman-support On Postman Web there has no empty tab.

**Comment by coder-free** [comment]
@postman-support Just now, I updated the postman native client, by click a button `downloaded upgrade` in `setting` menu. And there has a second empty tab appears. Now there has two empty tabs can't close.

**Comment by postman-staff** [comment]
@coder-free Thank you for the update.
Glad to know things are working fine on Postman Web!

We suspect that the issue wouldn't be resolved until the cache is cleared for Postman. May I know if you have followed the steps suggested previously for local installation? If not, could you follow them and let us know how it goes.


- Sign out of your account and
  - Quit application
- Clearing cache [as per the article here.](https://support.postman.com/hc/en-us/articles/4410785125271)
  - Simply put navigate to `~/library/Application support` and rename `Postman` to `Postman.old`
- Launch application and
  - Login back to your account

Let us know how it goes.

via @Raj-postman | Postman Support

**Comment by coder-free** [comment]
@postman-support Hi, I just sign out my account and relogin. These empty tabs are gone. Thanks.

**Comment by postman-staff** [comment]
@coder-free Awesome! :)
Glad to know that helped.

via @Raj-postman | Postman Support

**Comment by DieHardd** [comment]
> Hi @coder-free
> 
> Sorry to see you're encountering the issue.
> 
> We are aware of this case and the team is working on resolving the same. To let you know, the issue branched from the previous case where there were empty ids that can be cleared to resolve the same from your end. The resolution for the action would be:
> 
> * Signing out of your account
> * Clearing cache as [per the article here](https://support.postman.com/hc/en-us/articles/4410785125271)
> * Re-logging back to your account
> 
> These steps would completely clear out these cases here. Let us know how it goes and we will take it from there.
> 
> via @Raj-postman | Postman Support

My problem solved ty
